packages/react/src/components/Forms/DatePicker/index.scss

```diff
@@ -0,0 +1,287 @@
+@use 'src/utils';
+
+.c__date-picker {
+  position: relative;
+
+
+  &__wrapper {
+    border-radius: var(--c--components--forms-datepicker--border-radius);
+    border-width: var(--c--components--forms-datepicker--border-width);
+    border-color: var(--c--components--forms-datepicker--border-color);
+    border-style: var(--c--components--forms-datepicker--border-style);
+    display: flex;
+    align-items: center;
+    transition: border var(--c--theme--transitions--duration) var(--c--theme--transitions--ease-out);
+    padding: 0 0.75rem;
+    gap: 1rem;
+    color: var(--c--components--forms-datepicker--color);
+    box-sizing: border-box;
+    height: var(--c--components--forms-datepicker--height);
+    background-color: var(--c--components--forms-datepicker--background-color);
+    position: relative;
+    overflow: hidden;
+
+    &--clickable {
+      cursor: pointer;
+    }
+  }
+
+  &__inner {
+    flex-grow: 1;
+    display: flex;
+    justify-content: space-between;
+    user-select: none;
+    min-width: 0;
+
+    &--clickable {
+      cursor: pointer;
+    }
+
+    &__value {
+      display: flex;
+      font-size: var(--c--components--forms-datepicker--font-size);
+      overflow: hidden;
+      flex-grow: 1;
+
+      &__segment {
+        &--empty {
+          color: var(--c--components--forms-field--color);
+        }
+      }
+    }
+
+    &__action {
+      display: flex;
+      align-items: center;
+      height: auto;
+      width: 1.25rem !important;
```

Comment:
No possibility to increase selector specificity in order to prevent the use of `!important` ?